pkgs/by-name/pl/plakar/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildGoModule
+}:
+
+buildGoModule rec {
+  pname = "plakar";
+  version = "0.4.24-alpha";
+
+  src = fetchFromGitHub {
+    owner = "PlakarKorp";
+    repo = "plakar";
+    rev = "v${version}";
+    hash = "sha256-sAhBKIpPQL4n948ttICTUgvF7QbqqjYh+EdilEfu9XI=";
+  };
+
+  vendorHash = "sha256-AqhkX//8c7cc5LBBc4sjMFWPPKMjS7voM/X0lxlnHLU=";
+
+  meta = with lib; {
+    homepage = "https://github.com/PlakarKorp/plakar";
+    description = "plakar is a backup solution";
+    license = licenses.isc;
+    maintainers = with maintainers; [ heph2 ];
+    platforms = platforms.all;
+  };
```

Comment:
Movement seems to be going towards the removal of `with lib;` everywhere (#208242), FYI.